paymentsheet/src/main/java/com/stripe/android/common/ui/PrimaryButton.kt

```diff
@@ -41,18 +41,15 @@ internal fun PrimaryButton(
     modifier: Modifier = Modifier,
     isLoading: Boolean = false,
     displayLockIcon: Boolean = false,
-    overrideBackgroundColor: Color? = null,
-    overrideOnBackgroundColor: Color? = null,
-    overrideBorderColor: Color? = null,
 ) {
     // We need to use PaymentsTheme.primaryButtonStyle instead of MaterialTheme
     // because of the rules API for primary button.
     val context = LocalContext.current
-    val background = overrideBackgroundColor ?: Color(StripeTheme.primaryButtonStyle.getBackgroundColor(context))
-    val onBackground = overrideOnBackgroundColor ?: Color(StripeTheme.primaryButtonStyle.getOnBackgroundColor(context))
+    val background = Color(StripeTheme.primaryButtonStyle.getBackgroundColor(context))
+    val onBackground = Color(StripeTheme.primaryButtonStyle.getOnBackgroundColor(context))
     val borderStroke = BorderStroke(
         StripeTheme.primaryButtonStyle.shape.borderStrokeWidth.dp,
-        overrideBorderColor ?: Color(StripeTheme.primaryButtonStyle.getBorderStrokeColor(context))
+        Color(StripeTheme.primaryButtonStyle.getBorderStrokeColor(context))
```

Comment:
Looks like we weren't using these but was this intended to be removed with this PR?

Reply:
yeah! The only place these params were being used was for the remove button on this screen. This PR made it so that these params went from being used -> unused, which is why they're deleted here